guides/setup-dev-environment.md

```diff
@@ -308,6 +235,86 @@ Open two terminals one will be for shell access into minnaker-vm the other will
 
 Now magic happens.
 
+## Setup kubectl on host
+
+1. [minnaker-vm] Get your kubernetes config file
+
+    ```bash
+    kubectl config view --raw
+    ```
+
+    Example Output:
+    ```
+    apiVersion: v1
+    clusters:
+    - cluster:
+        certificate-authority-data: YOUR_CERT_HERE
+        server: https://127.0.0.1:6443
+    name: default
+    contexts:
+    - context:
+        cluster: default
+        namespace: spinnaker
+        user: default
+    name: default
+    current-context: default
+    kind: Config
+    preferences: {}
+    users:
+    - name: default
+    user:
+        password: YOUR_PASSWORD_HERE
+        username: admin
+    ```
+
+1. [host] Save the command output from above command `kubectl config view --raw` to `~/.kube/minnaker` on host machine
+
+1. [minnaker-vm] To get the IP of minnaker-vm
+
+   ```bash
+   cat /etc/spinnaker/.hal/public_endpoint
+   ```
+
+1. [host] Edit `~/.kube/minnaker` to have the IP address of the minnaker-vm
+    New File:
+    ```
+    apiVersion: v1
+    clusters:
+    - cluster:
+        certificate-authority-data: YOUR_CERT_HERE
+        server: https://192.168.64.6:6443
+    name: default
+    contexts:
+    - context:
+        cluster: default
+        namespace: spinnaker
+        user: default
+    name: default
+    current-context: default
+    kind: Config
+    preferences: {}
+    users:
+    - name: default
+    user:
+        password: YOUR_PASSWORD_HERE
+        username: admin
+    ```
+
+1. [host] Setup `kubectl` from HOST to check on the deploy
+
+    ```
+    export KUBECONFIG=~/.kube/minnaker
+    ```
+    or always specify `--kubeconfig ~/.kube/minnaker`
+    ```
+    kubectl --kubeconfig command
```

Comment:
really appreciate these edits, since we're modifying this - 
could you also make this a little clearer by making this a complete example

```
kubectl --kubeconfig ~/.kube/minnaker get pods -n spinnaker
```

Reply:
updated this to be a little more clear.